tests/base.py

```diff
@@ -38,6 +38,8 @@
 ROCEV2_UNSUPPORTED_DEVS = {MLNX_VENDOR_ID: [CX3Pro_MLNX_PART_ID,
                                             CX3_MLNX_PART_ID]}
 
+def has_hw_bug(vendor_id, vendor_part_id):
```

Comment:
The function name is too generic, let's change it to something like "has_roce_hw_bug"

Reply:
maybe has_roce_version_hw_bug() ?

Reply:
I think has_roce_hw_bug() makes more sense, I'll update the patch.